re.rizin.cutter.yml

```diff
@@ -0,0 +1,85 @@
+app-id: re.rizin.cutter
+command: cutter
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+separate-locales: false
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+build-options:
+  prepend-path: /usr/lib/sdk/llvm12/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+add-extensions:
+  re.rizin.cutter.plugin:
+    version: stable
+    directory: extensions
+    add-ld-path: lib
+    merge-dirs: plugins;bin;lib
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  #stolen from https://github.com/flathub/org.freecadweb.FreeCAD/blob/master/org.freecadweb.FreeCAD.yaml
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTS=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.6-src/pyside-setup-opensource-src-5.15.6.tar.xz
+        sha256: 171ee058d056599b8e9e8605fe3dae5dabdab7764285b21b28f64760a0f6779d
+      - type: shell
+        commands:
+          - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp # https://bugreports.qt.io/browse/PYSIDE-787
+          - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+  - name: rizin
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - --prefix=/app
+      - -Drizin_plugins=extensions/lib/rizin/plugins
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.4.1/rizin-src-v0.4.1.tar.xz 
+        sha256: 669d956b997820a36e423cf35d482aa99849254b9658cef6844459912dfdbfb8
+
+  - name: graphviz
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/2.49.0/graphviz-2.49.0.tar.gz
+        sha256: 17ade6189beade7c0784090f283c1f78f6b286ec38e3d2e927d3e42d625d0e8d
+
+  - name: cutter
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCUTTER_USE_BUNDLED_RIZIN=off
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=on
+      - -DCUTTER_ENABLE_PYTHON=on
+      - -DCUTTER_ENABLE_PYTHON_BINDINGS=on
+      - -DSHIBOKEN_EXTRA_OPTIONS="--include-paths=/app/include/qt5tmp" # https://bugreports.qt.io/browse/PYSIDE-787
+      - -DCUTTER_EXTRA_PLUGIN_DIRS=/app/extensions/plugins
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/cutter/releases/download/v2.1.2/Cutter-v2.1.2-src.tar.gz
+        sha256: 088351e280d158e07498098ffba355a6272c2e4faf4b350ac7778e369eb23fe1
+      - type: shell
+        commands:
+          - mkdir -p /app/extensions/plugins
```

Comment:
1. this doesn't match the extension point
2. this should be in the post-install instead.

Reply:
fixed